app/helpers/limit-array.ts

```
@@ -0,0 +1,5 @@
+import { helper } from '@ember/component/helper';
+
+export default helper(function limitArray([array, limit]: [any[], number]) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)